tests/config_test.go

```diff
@@ -46,3 +59,35 @@ func TestConfig(t *testing.T) {
 		assert.Equal(t, "cliptimeout: 120", out)
 	})
 }
+
+func TestMountConfig(t *testing.T) {
+	ts := newTester(t)
+	defer ts.teardown()
+
+	// we add a mount:
+	out, err := ts.run("init --store mnt/m1 --path " + ts.storeDir("m1") + " --rcs=noop " + keyID)
```

Comment:
`--rcs=noop` isn't supported anymore as of #1455. Please use `--storage=fs` instead.